presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -76,7 +76,7 @@
             "   CASE WHEN orderkey % 43 = 0 THEN null ELSE (CAST(discount AS DECIMAL(20, 8)), CAST(tax AS DECIMAL(20, 8))) END AS long_decimals, " +
             "   CASE WHEN orderkey % 11 = 0 THEN null ELSE (orderkey, partkey, suppkey) END AS keys, \n" +
             "   CASE WHEN orderkey % 41 = 0 THEN null ELSE (extendedprice, discount, tax) END AS doubles, \n" +
-            "   CASE WHEN orderkey % 13 = 0 THEN null ELSE ((orderkey, partkey), (suppkey,), CASE WHEN orderkey % 17 = 0 THEN null ELSE (orderkey, partkey) END) END AS nested_keys, \n" +
+            "   CASE WHEN orderkey % 13 = 0 THEN null ELSE ARRAY[ARRAY[orderkey, partkey], ARRAY[suppkey], CASE WHEN orderkey % 17 = 0 THEN null ELSE ARRAY[orderkey, partkey] END] END AS nested_keys, \n" +
```

Comment:
What is this change about? I don't think tuple and array has the same semantic meaning.

Reply:
This changes fixes the test failure for upgrading H2 from 1.4.197 to 1.4.199.
- This is an H2 query, not a Presto query. In H2 1.4.197, `(suppkey, )` represents a 1-element Row; in H2 1.4.199, this became a syntax error.
- This line of the H2 query is expected to match the Presto query on line 148 (which is Array, not Row), so it is actually more correct to write the H2 query in Arrays. Not planning to update all the other lines since improving the test is not the objective of this commit.
- H2 Rows are all unnamed, and is represented the same way as H2 Arrays. In the tests, H2 Rows are in the results are mapped to Arrays in `MaterializedRow`, and that's why the tests were passing even when the H2 query is selecting Rows while the Presto query is selecting Arrays.